pkg/apis/pipeline/v1beta1/pipelinerun_types_test.go

```diff
@@ -304,10 +304,12 @@ func TestPipelineRunGetServiceAccountName(t *testing.T) {
 		},
 	} {
 		for taskName, expected := range tt.saNames {
-			sa := tt.pr.GetServiceAccountName(taskName)
-			if expected != sa {
-				t.Errorf("%s: wrong service account: got: %v, want: %v", tt.name, sa, expected)
-			}
+			t.Run(tt.name, func(t *testing.T) {
+				sa := tt.pr.GetServiceAccountName(taskName)
+				if expected != sa {
```

Comment:
does `expected` here need to be shadowed inside the `if` now that it's wrapped in a `t.Run`?  Or is that only if `t.Parallel` has been called somewhere else?

Reply:
Only if `t.Parallel` is called inside the `t.Run` :wink: